### PR TITLE
[2.4]: add replicate end message

### PR DESCRIPTION
**Pull request by SimFG**
- issue: https://github.com/milvus-io/milvus/issues/37105
- pr: #360

### CODE REVIEW

File: proto/schema.proto [hunk 1]
```diff
@@ -77,6 +77,7 @@ message CollectionSchema {
   repeated FieldSchema fields = 4;
   bool enable_dynamic_field = 5; // mark whether this table has the dynamic field function enabled.
   repeated common.KeyValuePair properties = 6;
+  string dbName = 8;
```
Comment (czs007) [inline proto/schema.proto:80]: where 's 7 ?

Comment (SimFG) [reply proto/schema.proto:80]: ahhh

Comment (czs007) [reply proto/schema.proto:80]: please be carefull

**Review by czs007 — COMMENTED** [review]
where is 7

**Pull request merged**